.chloggen/k8sattribute-occasionaly-missing-attributes.yaml

```diff
@@ -0,0 +1,20 @@
+# Use this changelog template to create an entry for release notes.
+# If your change doesn't affect end users, such as a test fix or a tooling change,
+# you should instead start your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Fix to tests should not be considered a bug fix. The changelog entry is not needed here

Reply:
Done @dmitryax